obal/data/roles/build_package/tasks/waitrepo.yml

```diff
@@ -0,0 +1,14 @@
+- when: build_package_build_system == 'koji' and build_package_waitrepo|bool
```

Comment:
Nice! We should do this more often, feels more like a traditional `if-then` relationship to me.